guide/anatomy.rst

```diff
@@ -111,25 +111,38 @@ Other motors and sensors have similar conventions.
 Robot drivetrain control
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-For standard types of drivetrains (2 or 4 wheel, and mecanum), you'll want to
-use the :class:`.RobotDrive` class to control the motors instead of writing
-your own code to do it. When you create a RobotDrive object, you either specify
-which PWM channels to automatically create a motor for::
+For standard types of drivetrains (2 or 4 wheel, mecanum, kiwi), you'll want to
+use the various included class to control the motors instead of writing
+your own code to do it. For most standard drivetrains, you'll want to use one
+of three classes:
 
-	self.robot_drive = wpilib.RobotDrive(0,1)
+* :class:`wpilib.drive.DifferentialDrive` for differential drive/skid-steer drive platforms such as 2 or 4 wheel platforms, the Kit of Parts drive base, "tank drive", or West Coast Drive.
+* :class:`wpilib.drive.KilloughDrive` for Killough (Kiwi) triangular drive platforms.
+* :class:`wpilib.drive.MecanumDrive` for mecanum drive platforms.
 
-Or you can pass in motor controller instances::
+For example, when you create a :class:`.DifferentialDrive` object, you can pass in motor controller instances::
+
+    l_motor = wpilib.Talon(0)
+    r_motor = wpilib.Talon(1)
+    self.robot_drive = wpilib.drive.DifferentialDrive(l_motor, r_motor)
+
+Or you can pass in motor controller groups to use more than one controller per side::
+
+    self.frontLeft = wpilib.Spark(1)
+    self.rearLeft = wpilib.Spark(2)
+    self.left = wpilib.SpeedControllerGroup(self.frontLeft, self.rearLeft)
+
+    self.frontRight = wpilib.Spark(3)
+    self.rearRight = wpilib.Spark(4)
+    self.right = wpilib.SpeedControllerGroup(self.frontRight, self.rearRight)
+
+    self.drive = wpilib.drive.DifferentialDrive(self.left, self.right)
 
-	l_motor = wpilib.Talon(0)
-	r_motor = wpilib.Talon(1)
-	self.robot_drive = wpilib.RobotDrive(l_motor, r_motor)
-	
 Once you have one of these objects, it has various methods that you can use
 to control the robot via joystick, or you can specify the control inputs
 manually.
 
-.. seealso:: Documentation for the :class:`wpilib.robotdrive.RobotDrive`
-             object, and the FIRST WPILib Programming Guide.
+.. seealso:: Documentation for the :class:`wpilib.drive.DifferentialDrive`, :class:`wpilib.drive.KilloughDrive`, and :class:`wpilib.drive.MecanumDrive` objects, and the FIRST WPILib Programming Guide.
```

Comment:
I think it might be more appropriate to link to the `wpilib.drive` package, rather than list each individual class, as I would expect more classes to be added in future.

Reply:
Changed!